assistant_dists/dream_ocean/dev.yml

```diff
@@ -144,4 +144,10 @@ services:
       - "./annotators/personality_detection:/src"
     ports:
       - 8026:8026
+  emotion-detection:
+    volumes:
+      - "./annotators/emotion_detection:/src/drm"
+      - "/home/dikanskiy/emotion_test:/src/datafiles"
```

Comment:
nope this is illegal. Remove this line

Reply:
removed (but one would still need to add some mapping to /src/datafiles or have a videofile in this directory inside the container in order for the annotator to work)

Reply:
why can't you get a file as images (path from attirbutes of the current utterance and the file itself from FILES container)?

Reply:
the input to this service is a video + text so I used a test mounted folder as we discussed with @oserikov